src/exoscale/lingo/utils.cljc

```diff
@@ -12,5 +12,5 @@
    :cyan "\u001b[36;1m"
    :reset "\u001b[0m"})
 
-(defn color [s color]
-  (str (color colors) s (:reset colors)))
+(defn color [s color-kw]
```

Comment:
Other kinds of shadowing don't matter as much, but shadowing the surrounding defn's name itself is particularly dangerous, as it can create unintended recursion over time.

Reply:
right, good catch. thanks